docs/pages/_meta.js

```diff
@@ -4,8 +4,10 @@ export default {
   "what-is-mud": "What is MUD?",
   store: "Store",
   world: "World",
+  "client-side": "MUD Client",
```

Comment:
nit: it'd be nice to stick with sentence-case for these (I made some changes in this direction in #857)

(learned at Stripe that sentence-case is generally easier to read and write)